### PR TITLE
Unifying words in recovery draft

### PR DESCRIPTION
**Pull request by kazu-yamamoto**
This should resolve some parts of #3813.

### CODE REVIEW

File: draft-ietf-quic-recovery.md [hunk 1]
```diff
@@ -518,7 +518,7 @@ or when Initial or Handshake keys are discarded. This ensures the PTO is always
 set based on the latest RTT information and for the last sent packet in the
 correct packet number space.
 
-When ack-eliciting packets in multiple packet number spaces are in flight,
+When ack-eliciting packets in multiple packet number spaces are in-flight,
```
Comment (marten-seemann) [inline draft-ietf-quic-recovery.md:521]: I think "in flight" is more correct here.

File: draft-ietf-quic-recovery.md [hunk 2]
```diff
@@ -615,7 +615,7 @@ a packet containing unacknowledged CRYPTO data earlier than the PTO expiry,
 subject to address validation limits; see Section 8.1 of {{QUIC-TRANSPORT}}.
 
 Peers can also use coalesced packets to ensure that each datagram elicits at
-least one acknowledgement.  For example, clients can coalesce an Initial packet
+least one acknowledgement.  For example, a client can coalesce an Initial packet
```
Comment (marten-seemann) [inline draft-ietf-quic-recovery.md:618]: The previous sentence uses a plural. Why change one but not the other?

File: draft-ietf-quic-recovery.md [hunk 3]
```diff
@@ -657,7 +657,7 @@ send, the sender SHOULD send a PING or other ack-eliciting frame in a single
 packet, re-arming the PTO timer.
 
 Alternatively, instead of sending an ack-eliciting packet, the sender MAY mark
-any packets still in flight as lost.  Doing so avoids sending an additional
+any packets still in-flight as lost.  Doing so avoids sending an additional
```
Comment (marten-seemann) [inline draft-ietf-quic-recovery.md:660]: Again here, I think "in flight" reads better.

File: draft-ietf-quic-recovery.md [hunk 4]
```diff
@@ -697,7 +697,7 @@ When packet protection keys are discarded (see Section 4.10 of {{QUIC-TLS}}),
 all packets that were sent with those keys can no longer be acknowledged because
 their acknowledgements cannot be processed anymore. The sender MUST discard
 all recovery state associated with those packets and MUST remove them from
-the count of bytes in flight.
+the count of bytes in-flight.
```
Comment (marten-seemann) [inline draft-ietf-quic-recovery.md:700]: Same here.

**Comment by kazu-yamamoto** [comment]
@marten-seemann If you wish, I will split this PR into small ones.

File: draft-ietf-quic-recovery.md (hunk 1 above)
Comment (kazu-yamamoto) [reply draft-ietf-quic-recovery.md:521]: I might be too strict but "in flight" is not defined anywhere. When I looked into stuff relating to "in-flight", I actually had to search both "in-flight" and "in flight". It was inconvenience.

File: draft-ietf-quic-recovery.md (hunk 2 above)
Comment (kazu-yamamoto) [reply draft-ietf-quic-recovery.md:618]: This is followed by "a server". Why are both singular and plural used in the same sentence?

File: draft-ietf-quic-recovery.md (hunk 1 above)
Comment (martinthomson) [reply draft-ietf-quic-recovery.md:521]: This is a quirk in English.  The hyphen is only used when "in-flight" is a modifier for some other term, like "in-flight packets".  https://en.wikipedia.org/wiki/English_compound#Hyphenated_compound_modifiers

**Comment by kazu-yamamoto** [comment]
I have dropped the commit of "in flight".

**Comment by kazu-yamamoto** [comment]
I also changed `#` into `//`.

File: draft-ietf-quic-recovery.md [hunk 5]
```diff
@@ -615,7 +615,7 @@ a packet containing unacknowledged CRYPTO data earlier than the PTO expiry,
 subject to address validation limits; see Section 8.1 of {{QUIC-TRANSPORT}}.
 
 Peers can also use coalesced packets to ensure that each datagram elicits at
```
Comment (martinthomson) [inline draft-ietf-quic-recovery.md:617]:
```suggestion
Endpoints can also use coalesced packets to ensure that each datagram elicits at
```

File: draft-ietf-quic-recovery.md [hunk 6]
```diff
@@ -145,10 +145,10 @@ of frames contained in a packet affect recovery and congestion control logic:
   performance of the QUIC handshake and use shorter timers for
   acknowledgement.
 
-* Packets containing frames besides ACK or CONNECTION_CLOSE frames count toward
+* Packets containing frames besides ACK or CONNECTION_CLOSE frames count towards
   congestion control limits and are considered in-flight.
 
-* PADDING frames cause packets to contribute toward bytes in flight without
+* PADDING frames cause packets to contribute towards bytes in flight without
```
Comment (martinthomson) [inline draft-ietf-quic-recovery.md:151]:
```suggestion
* PADDING frames cause packets to contribute toward bytes in flight without
```

File: draft-ietf-quic-recovery.md [hunk 7]
```diff
@@ -145,10 +145,10 @@ of frames contained in a packet affect recovery and congestion control logic:
   performance of the QUIC handshake and use shorter timers for
   acknowledgement.
 
-* Packets containing frames besides ACK or CONNECTION_CLOSE frames count toward
+* Packets containing frames besides ACK or CONNECTION_CLOSE frames count towards
```
Comment (martinthomson) [inline draft-ietf-quic-recovery.md:148]: I think that the original is correct.

```suggestion
* Packets containing frames besides ACK or CONNECTION_CLOSE frames count toward
```

File: draft-ietf-quic-recovery.md (hunk 2 above)
Comment (martinthomson) [reply draft-ietf-quic-recovery.md:618]: I think that "a client" and "a server" are best here.  The previous sentence is general, this is an example and therefore specific.

That said, we are almost certainly hugely inconsistent about addressing requirements to "endpoints" (plural) or "an endpoint" (singular).

**Comment by kazu-yamamoto** [comment]
@martinthomson The commit for "towards" has been dropped.

**Comment by martinthomson** [comment]
Thanks.  BTW, that [toward/towards](https://www.grammarly.com/blog/toward-towards/) thing is an another odd English quirk.

**Pull request merged**